### ISSUE TITLE
Manage multiple events on save pipeline (PART I)

### ISSUE THREAD

**Issue by zeppaman** [arduosoft/RawCMS#165]
Now we have a "PreSave" script that alter data before it is written on database. The evolution of this is managing all possible events.

# Event list

- PreSave
- PostSave
- PreDelete
- PostDelete

In the future, we may have more events, for now just start with that.

# On UI
the vertical tab list will contain one script for each event. This list can be fixe for now.

# Data model
Now we have a field for the only event. Keeping all in this way may be confusional for lot of events. So we nedd to add some inner object like:
```
{
  MAIN OBJECT
  Events
 {
    "PreSave": "XXX"
    "PostSave": "XXX"
 }
}
```
Basically an entry in form <event name>: javascript code insite an "Event" subsection
